src/components/text.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { ReactElement } from "react"
+
+interface TextProps {
+  as?:
+    | "h1"
+    | "h2"
+    | "p"
+    | "h3"
+    | "h3Link"
+    | "h2about"
+    | "pAbout"
+    | "pAuthor"
+    | "h2episode"
+    | "pEpisode"
+
+  children: string | any
+}
+
+export default function Text({
+  children,
+  as,
+
+  ...props
+}: TextProps) {
+  let classNamesVariant = ""
+
+  if (as === "h1") {
+    classNamesVariant =
+      "mb-0 text-gray-700 text-6xl font-merri font-bold leading-snug"
```

Comment:
it's better if you
```
return <h1 className="mb-0 text-gray-700 text-6xl font-merri font-bold leading-snug">{children}</h1>
```
Because it's important for the SEO
https://www.socialmediatoday.com/news/8-of-the-most-important-html-tags-for-seo/574987/

Same for the others tags bellow